docs/root/configuration/http/http_filters/grpc_json_transcoder_filter.rst

```diff
@@ -82,7 +82,10 @@ as its output message type. The implementation needs to set
 (which sets the value of the HTTP response `Content-Type` header) and
 `data <https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto#L71>`_
 (which sets the HTTP response body) accordingly.
-
+Please note that when gRPC response streaming is used HTTP response header `Content-Type` will be set
```

Comment:
Multiple Httpbody can be send by the gRPC server in the server streaming case. In this case, HTTP response header `Content-Type` will use the content-type from the first Httpbody.

Reply:
Rephrased.